fish_prompt.fish

```diff
@@ -213,6 +213,10 @@ function prompt_status -d "the symbols for a non zero exit status, root and back
     end
 end
 
+function available -a name -d "Check if a function or program is available."
```

Comment:
Askirhg because its already present on of and this could break omf in the future for this theme users.